### PR TITLE
Fix ability to define custom page size as per README

### PR DESCRIPTION
**Pull request by icidasset**
So, the readme said I could use a custom `pageSize`, but it doesn't actually work.
This should fix it. Wrote this rather quickly, so feedback is welcome 😄

### CODE REVIEW

**Comment by codecounselor** [comment]
I wonder if my recent changes broke this? Was it working before?  In any event I think we should be able to merge it soon.  My https://github.com/fraserxu/electron-pdf/pull/109 PR will likely cause merge conflicts with these changes.  Would you be willing to merge them in on top of these changes?

File: index.js [hunk 1]
```diff
@@ -113,7 +113,20 @@ var PDFExporter = function (input, output, argv) {
       'Tabloid': {x: pdfToPixels(11), y: pdfToPixels(17)}
     }
 
-    var pageDim = pageDimensions[argv.pageSize]
+    var pageDim
+
+    if (typeof argv.pageSize === 'object') {
+      var xInches = argv.pageSize.width / 25400
```
Comment (icidasset) [inline index.js]: Ah sure. 1 inch = 25400 microns

**Comment by icidasset** [comment]
I only started using this library yesterday, so not sure what you're talking about 😄 
Let me know when you merge that PR, then I'll rebase and adjust.

**Comment by codecounselor** [comment]
haha, well thanks for the PR! I did the merge so if you rebase we can get your changes in.  Feel free to contribute a unit test, there is now support for running Jasmine tests.  In a PR very soon I'll be switching to Ava if you prefer to use that.

**Comment by icidasset** [comment]
No problem! Will change in a bit. I prefer Ava, so I'll wait for that ;)

**Comment by icidasset** [comment]
Ok, rebased and improved 😌

**Comment by codecounselor** [comment]
I'm pretty new to the project, so I will let @fraserxu cut a new release if you need your patch in NPM soon.  

With all of the changes I've introduced I'm thinking maybe we should bump the minor revision if the goal is to follow [semver](http://semver.org/).

**Comment by icidasset** [comment]
Definitely follow semver, and thanks 🙏 😄

File: lib/charge.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ if (argv.version) {
   process.exit(0)
 }
 
+if (argv.pageSize && argv.pageSize[0] === '{') {
```
Comment (codecounselor) [inline lib/charge.js]: I noticed while merging some of my own pull requests that having this in charge.js will work fine if electron-pdf is invoked using the CLI.  But if you want to use it as a module in a program then charge.js is not used.  This should be moved somewhere inside index.js.  

Maybe you can do that when you write some tests?

**Pull request merged**